src/drive/mobile/lib/reporter.js

```diff
@@ -16,41 +14,36 @@ const getAnalyticsUrl = () => {
 export const ANALYTICS_URL = getAnalyticsUrl()
 
 export const getReporterConfiguration = () => ({
-  shouldSendCallback: () => isEnabled,
+  shouldSendCallback: true,
   environment: __DEVELOPMENT__ ? 'development' : 'production'
 })
 
-export const configureReporter = enable => {
-  isEnabled = enable
+export const configureReporter = () => {
   Raven.config(ANALYTICS_URL, getReporterConfiguration()).install()
 }
 
-export const logException = err => {
+export const logException = (err, extraContext = null) => {
   return new Promise(resolve => {
-    Raven.captureException(err)
+    extraContext
+      ? Raven.captureException(err, { extra: extraContext })
+      : Raven.captureException(err)
     console.warn('Raven is recording exception')
     console.error(err)
     resolve()
   })
 }
 
-const logMessage = (message, serverUrl, level = 'info', force) => {
+const logMessage = (message, serverUrl, level = 'info') => {
```

Comment:
good move to remove this weird parameter: if I call `logMessage`, I want it to log a message. Using a force parameter let me believe `logMessage` may be called `pleaseAreYouAbleToLogMessage` ;)